src/utils.js

```diff
@@ -66,7 +66,9 @@ function wrapContent($parent, $node) {
 function toClassName(text) {
   return text
     .toLowerCase()
-    .replace(/[^0-9a-z]/gi, '-');
+    .replace(/[^0-9a-z]+/gi, '-')
+    .replace(/^-+/g, '')
+    .replace(/-+$/g, '');
```

Comment:
btw, I also fixed the `toClassname` to make it nicer. but it now has errors in the diff tests:

![image](https://user-images.githubusercontent.com/917628/117945919-c5f3b600-b349-11eb-9f48-8bae6514bbc4.png)

![image](https://user-images.githubusercontent.com/917628/117946000-d9068600-b349-11eb-8cd8-de6540c8a92b.png)

(the red is the **new** one)

@davidnuescheler is this a problem?